parquet-avro/src/main/java/org/apache/parquet/avro/AvroRecordConverter.java

```diff
@@ -236,6 +236,8 @@ private static Converter newConverter(Schema schema, Type type,
       GenericData model, Class<?> knownClass, ParentValueContainer setter) {
     LogicalType logicalType = schema.getLogicalType();
     Conversion<?> conversion;
+    Class<?> datumClass;
```

Comment:
Why move this up? I prefer to have it at the previous place.

Reply:
With the `switch` statement, the scope changed and therefore I could not keep it where it was, I had to make it available to a wider scope.  I have made each their own scope now.  Please check the change.  Thanks for the review!

Reply:
Thanks, looks good now